lib/lazy_load_indexed_stack.dart

```diff
@@ -45,26 +45,33 @@ class LazyLoadIndexedStack extends StatefulWidget {
 class _LazyLoadIndexedStackState extends State<LazyLoadIndexedStack> {
   late final List<bool> _alreadyLoaded;
```

Comment:
Thanks to this modification, the variable `_alreadyLoaded` is no longer needed, so I thought the whole code could look something like this, but guess what?

I named the function `_loadedChildren` as `_initialChildren`.

```dart
class _LazyLoadIndexedStackState extends State<LazyLoadIndexedStack> {
  late List<Widget> _children;
  final _stackKey = GlobalKey();

  @override
  void initState() {
    super.initState();

    _children = _initialChildren();
  }

  @override
  void didUpdateWidget(final LazyLoadIndexedStack oldWidget) {
    super.didUpdateWidget(oldWidget);

    _children[widget.index] = widget.children[widget.index];
  }

  @override
  Widget build(final BuildContext context) {
    return IndexedStack(
      key: _stackKey,
      index: widget.index,
      children: _children,
      alignment: widget.alignment,
      textDirection: widget.textDirection,
      sizing: widget.sizing,
    );
  }

  List<Widget> _initialChildren() {
    return widget.children.asMap().entries.map((entry) {
      final index = entry.key;
      final childWidget = entry.value;
      return index == widget.index ? childWidget : widget.unloadWidget;
    }).toList();
  }
}
```